scripts/run-iroha-dev.sh

```diff
@@ -1,8 +1,14 @@
 #!/bin/bash
 
-USER=iroha
 USERID=$(id -u $(whoami))
 
+
```

Comment:
add check that `IROHA_HOME` is set.

if not set -- print error message and exit with non-0 exit code

Reply:
We assign IROHA_HOME in this script